src/components/ChatBotInput/ChatBotInput.tsx

```diff
@@ -7,6 +7,7 @@ import { isDesktop } from "../../services/Utils";
 import { useBotOptions } from "../../context/BotOptionsContext";
 
 import "./ChatBotInput.css";
+import React from "react";
```

Comment:
This should come at the very top of the file.

Reply:
fixed